### PR TITLE
fix: Custom Cypress tests

### PR DESCRIPTION
**Pull request by lucasfp13**
## What's the purpose of this pull request?

This PR intends to fix some details regarding stores' custom Cypress tests.

### Starters Deploy Preview

https://github.com/vtex-sites/starter.store/pull/237

### CODE REVIEW

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 1]
```diff
@@ -29,10 +29,11 @@ Learn how to add custom integration tests in the following steps.
 
 - Please note that this feature is experimental.
 
-- Ensure that you are using `@faststore/core` and `@faststore/cli` version 2.1.56 or above. If you have a version that is lower than this requirement, change the following:
+- Ensure that you are using `@faststore/core` version 2.1.56 or above and `@faststore/cli` version 2.2.6 or above. If you have a version that is lower than this requirement, change the following:
   1. In your store code, open the `package.json` file.
-  2. Navigate to the `@faststore/core` and `@faststore/cli` and and update them to be at least `2.1.56`.
-  3. Run `yarn dev` to update your changes.
+  2. Navigate to the `@faststore/core` and and update them to be at least `2.1.56`.
```
Comment (hellofanny) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:34]:
```suggestion
  2. Navigate to the `@faststore/core` and and update it to be at least `2.1.56`.
```

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 2]
```diff
@@ -29,10 +29,11 @@ Learn how to add custom integration tests in the following steps.
 
 - Please note that this feature is experimental.
 
-- Ensure that you are using `@faststore/core` and `@faststore/cli` version 2.1.56 or above. If you have a version that is lower than this requirement, change the following:
+- Ensure that you are using `@faststore/core` version 2.1.56 or above and `@faststore/cli` version 2.2.6 or above. If you have a version that is lower than this requirement, change the following:
```
Comment (hellofanny) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:32]:
```suggestion
- Ensure that you are using `@faststore/core` version `2.1.56` or above and `@faststore/cli` version `2.2.6` or above. If you have a version that is lower than this requirement, change the following:
```

File: packages/cli/src/commands/test.ts [hunk 3]
```diff
@@ -43,6 +43,12 @@ async function storeTest() {
 
 export default class Test extends Command {
   async run() {
+    if (!existsSync(tmpDir)) {
```
Comment (gvc) [inline packages/cli/src/commands/test.ts:46]: Why did we stop generating if the `.faststore` folder wasn't there? This seems to be more annoying and error-prone than beneficial to our users.

Comment (lucasfp13) [reply packages/cli/src/commands/test.ts]: We will copy the Cypress files once again when running the test command 🤝

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 4]
```diff
@@ -30,11 +30,11 @@ Learn how to add custom integration tests in the following steps.
 
 - Refer to [Cypress documentation](https://docs.cypress.io/guides/overview/why-cypress) for more information on this testing tool.
 
-- Ensure that you are using `@faststore/core` and `@faststore/cli` version 2.1.56 or above. If you are using a previous version, take the following steps:
+- Ensure that you are using `@faststore/core` version `2.1.56` or above and `@faststore/cli` version `2.2.6` or above. If you have a version that is lower than this requirement, change the following:
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:33]:
```suggestion
- Ensure you are using `@faststore/core`  `2.1.56` version or above and `@faststore/cli` version `2.2.6` or above. If you have versions lower than these requirements, update them as follows:
```

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 5]
```diff
@@ -30,11 +30,11 @@ Learn how to add custom integration tests in the following steps.
...
   1. In your storefront project, open the `package.json` file.
-  2. Navigate to the `@faststore/core` and `@faststore/cli` dependencies and and update their versions to be at least `2.1.56`.
-  3. Run `yarn install` to apply these changes.
-
+  2. Navigate to the `@faststore/core` and update it to be at least `2.1.56`.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:35]:
```suggestion
  2. Navigate to the `@faststore/core` and update it to at least `2.1.56`.
```

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 6]
```diff
@@ -30,11 +30,11 @@ Learn how to add custom integration tests in the following steps.
...
+  3. Navigate to the `@faststore/cli` and update it to be at least `2.2.6`.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:36]:
```suggestion
  3. Navigate to the `@faststore/cli` and update it to at least `2.2.6`.
```

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 7]
```diff
@@ -30,11 +30,11 @@ Learn how to add custom integration tests in the following steps.
...
+  4. Run `yarn dev` to update your changes.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:37]:
```suggestion
  4. Run `yarn dev` to apply these changes.
```

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 8]
```diff
@@ -51,14 +51,15 @@ Learn how to add custom integration tests in the following steps.
   }
 ```
 
----
-
 ## Step 2: Integrating custom Cypress tests into your FastStore project
 
+Just like any other project using Cypress, all standard Cypress folders and configuration files are supported. (see official Cypress configuration guide [here](https://docs.cypress.io/guides/references/configuration))
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:56]:
```suggestion
Like any other project using Cypress, you'll follow standard practices, including setting up standard Cypress folders and configuration files. This allows you to seamlessly add and execute your custom integration tests within your project. For further details on Cypress configuration, refer to the official [Cypress configuration](https://docs.cypress.io/guides/references/configuration) guide.
```

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 9]
```diff
@@ -51,14 +51,15 @@ Learn how to add custom integration tests in the following steps.
...
 1. In the root directory of your storefront project, create a new folder named `cypress`.
-2. Add your custom integration tests within the `cypress` folder. 
-3. Open a pull request with the changes you have made.
-4. Within the pull request, navigate to **Checks > Integration Tests** and check the **Details** section.
+2. Add your custom integration tests within the `cypress/integration` folder.
+3. With the local server running, run `yarn test` to open the Cypress app and to check your tests execution.
+4. Open a pull request with the changes you have made.
+5. Within the pull request, navigate to **Checks > Integration Tests** and check the **Details** section.
 
 ![checks-integration-tests](https://vtexhelp.vtexassets.com/assets/docs/src/custom-test-checks___583ac9b78e43ebdafde93a735a47d61b.png)
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:65]: I think the last step (_Review the results to check the execution status of your custom integration tests._) was supposed to be number "6".

**Review by Mariana-Caetano — APPROVED** [review]

**Review by hellofanny — APPROVED** [review]
🚀

**Pull request merged**